ApollosProject/apollos-prototype
# Add Back Navigation to Auth Screens

## DESCRIPTION

### What does this PR do, or why is it needed?

Adds back buttons to the Verification and Passwords screens pictured below. This will allow users to navigate back should they wish to choose another path or enter a different phone number.

### What design trade-offs/decisions were made?

@kevin-birdy Not so much a design decision as a callout; the Verification screen looks a little funky because of the stacking of the button on top of the logo on top of the title, so...

### How do I test this PR?

1. Launch the app
3. Choose to use your email to sign in
3. Tap the back button; you should return to the previous screen
4. Enter your phone number
3. Tap the back button; you should return to the previous screen

### What automated tests did you write?

## SCREENSHOTS:

||IOS|Android|
|---|---|---|
|Verification|<img width="569" alt="Screen Shot 2019-10-09 at 4 40 49 PM" src="https://user-images.githubusercontent.com/29125070/66519177-c7e1fd00-eab4-11e9-9a2d-9064f96e51ed.png">|<img width="545" alt="Screen Shot 2019-10-09 at 4 46 51 PM" src="https://user-images.githubusercontent.com/29125070/66519191-d16b6500-eab4-11e9-8063-5c899ad2ccb7.png">|
|Password|<img width="569" alt="Screen Shot 2019-10-09 at 4 41 26 PM" src="https://user-images.githubusercontent.com/29125070/66519206-d92b0980-eab4-11e9-8c0f-81d5bd2398b6.png">|<img width="545" alt="Screen Shot 2019-10-09 at 4 46 37 PM" src="https://user-images.githubusercontent.com/29125070/66519233-ed6f0680-eab4-11e9-849a-b0418f59370f.png">|

## TODO:

- [x] Closes #1109 

## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
